deploy/sre-prometheus/100-kube-controller-manager-crashlooping.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-managed-kube-controller-manager-crashlooping
+    role: alert-rules
+  name: sre-managed-kube-controller-manager-crashlooping
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-managed-kube-controller-manager-crashlooping
+    rules:
+    - alert: KubeControllerManagerCrashlooping
```

Comment:
Just noting that a lot of our SRE alerts include a `SRE` suffix to indicate they are of SRE-defined origin. There is no hard requirement around this but it's a loose convention we seemed to have followed.

Reply:
Thank you That is an excellent point. :) I will fix that.

Reply:
Fixed in latest changes.